qa/rpc-tests/mergetoaddress_mixednotes.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# Copyright (c) 2019 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import sys; assert sys.version_info < (3,), ur"This script does not run under Python 3. Please use Python 2.7.x."
+
+from decimal import Decimal
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, get_coinbase_address, \
+    initialize_chain_clean, start_nodes, wait_and_assert_operationid_status
+
+
+class MergeToAddressMixedNotes(BitcoinTestFramework):
+    def setup_nodes(self):
+        return start_nodes(4, self.options.tmpdir, [[
+            '-nuparams=5ba81b19:100',  # Overwinter
+            '-nuparams=76b809bb:100',  # Sapling
+            '-experimentalfeatures', '-zmergetoaddress'
+        ]] * 4)
+
+    def setup_chain(self):
+        print("Initializing test directory " + self.options.tmpdir)
+        initialize_chain_clean(self.options.tmpdir, 4)
+
+    def run_test(self):
```

Comment:
Please add a test case for the error when `ANY_SPROUT` and `ANY_SAPLING` are both given, but there are not notes of both types.

Reply:
As it stands now, _I think_ this will end up covering the same code as the test added in the `mergetoaddress_helper.py`, but I will go ahead an add an assertion for when we have only Sprout notes and try to use `ANY_SPROUT` and `ANY_SAPLING`, and a similar one for a wallet with only Sapling notes.